JAVASCRIPT GAME/Jump Game/Readme.md

```diff
@@ -0,0 +1,13 @@
+# Jump-Game
+![image](https://user-images.githubusercontent.com/64858573/124364202-8bc5d700-dc5d-11eb-94df-5aa4311c1a63.png)
+
+* This web app allows you to play the jump game where you have to make box jump and safe himself from the small boxes coming in down and up direction . If the box touches the small box, the player lose. This is a fun game to play and a simple version of the one available in chrome when you are offline. Also, a good game to get started with Javascript.
+
+## Technologies used:
+* Html
+* Css
+* JS
+## How to set-up:
+* Clone this repo
+* Open it in any Text Editor 
+* Enjoy it !!!
```

Comment:
Use a markdown beautifier tool.

Here use this link https://codebeautify.org/markdown-formatter